continuous_integration/environment-3.10-dev.yaml

```diff
@@ -25,7 +25,7 @@ dependencies:
 - pytest-xdist
 - pytest
 - python=3.10
-- rust>=1.62.1
+- rust=1.62.1
```

Comment:
Curious why we want to pin as opposed to just having a lower bound?

Reply:
I figured we would want our CI running against the same version of Rust we use to compile the conda packages, rather than potentially picking up new versions

Reply:
Ok. The main trick is keeping this up-to-date when it is pinned tightly like this.